README.md

```diff
@@ -1,57 +1,10 @@
-# Northcoders News API
```

Comment:
Please tell the user to run `npm install` too otherwise they'll encounter a load of bugs if they try to use the api